README.md

```diff
@@ -65,5 +82,15 @@ a Gemfile cluttering up your repo. In that case, you can (probably) use
 this command in the Travis config above:
 
 ```bash
-rbenv global 2.3.1 && gem install danger --version '~> 5.0' && danger
+rbenv global 2.3.1 && gem install moj-danger --version && danger
 ```
+
+## Fixing your commits
+
+To keep Danger happy, you will need to amend commits that you have already pushed.
+
+This [Github article](https://help.github.com/articles/changing-a-commit-message/)
+describes a process for doing this. But, please use `--force-with-lease` instead
+of `--force` when pushing your amendments. See [this article](https://developer.atlassian.com/blog/2015/04/force-with-lease/)
```

Comment:
“use `--force-with-lease` instead of `--force`” might make more sense as the link text for the Atlassian article.